tmt/steps/provision/testcloud.py

```diff
@@ -93,6 +93,18 @@ def import_testcloud() -> None:
     - name: {user_name}
       ssh_authorized_keys:
         - {public_key}
+systemd:
+  units:
+    - name: ssh_root_login.service
+      enabled: true
+      contents: |
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/sed -i \
+                  "s|^PermitRootLogin no$|PermitRootLogin yes|g" \
+                  /etc/ssh/sshd_config
+        [Install]
+        WantedBy=multi-user.target
```

Comment:
Is it guaranteed that `sshd` starts (or reloads) after this unit executes?

Reply:
Not quite sure, how about adding `Before=sshd.service`?